tasks/libs/github_actions_tools.py

```diff
@@ -123,13 +125,13 @@ def follow_workflow_run(run):
         if status == "completed":
             return conclusion, run_url
         else:
-            print(f"Workflow still running... ({minutes}m)")
```

Comment:
I do not think you need this change since you add `interval` to `minutes` on each interation

Reply:
Indeed, fixed

Reply:
You fixed it even before my message was sent ahah